README.md

```diff
@@ -52,26 +52,26 @@ gulp.task('templates', function() {
 
 All options supported by the [Jade API](http://jade-lang.com/api/) are supported
```

Comment:
@thisboyiscrazy That should be fine for now.  I'll go ahead and change it in the future if I don't notice (because it should redirect too).